src/restapi/manage.rs

```diff
@@ -0,0 +1,110 @@
+use actix_web::{http, web, Responder};
+//use web::Json;
+use futures::future::OptionFuture;
+
+use super::{state, types};
+use crate::matchengine::server::matchengine::{self, matchengine_client::MatchengineClient};
+use crate::storage;
+
+pub mod market {
+
+    use super::*;
+
+    pub async fn add_assets(req: web::Json<types::NewAssetReq>, app_state: web::Data<state::AppState>) -> impl Responder {
+        let assets_req = req.into_inner();
+
+        for asset in &assets_req.assets {
+            log::debug!("Add asset {:?}", asset);
+            if let Err(e) = storage::config::persist_asset_to_db(&app_state.db, asset, assets_req.force_update).await {
```

Comment:
"add_assets"  is  public api.

That means everyone can force_update if they wish.

I suggest  hardcode "force_update" as false here

Reply:
Ok. But by doing that wwe will have no way to update an asset except update the db rows manually.
 
And, should we consider this api (adding assets) a public one? These routes under "manage" scope all are applied with the same security policy: With the "manage_endpoint" option unset, an restapi instance can not access both matching engine and write into database. So we can only configure instances which run in a safe space with this option to help managing market entries.

Reply:
> should we consider this api (adding assets) a public one?


It's one of our features --- permisionless listing.

But you make a good point, we need a entry point for administator to force update. This can be left as future plan.